README.md

```diff
@@ -176,3 +176,11 @@ The SavedModel will be saved in a timestamped directory under `/tmp/mnist_saved_
 ## Getting predictions with SavedModel
 
 Use [`saved_model_cli`](https://www.tensorflow.org/guide/saved_model#cli_to_inspect_and_execute_savedmodel) to inspect and execute the SavedModel.
+
+## Getting predictions using a script
+1. Clone down this repository
+1. Create a virtual environment (i.e. using `virtualenv`) in the root of the cloned directory
+1. Run `pip install -r requirements.txt`
+1. Run `python serving_rest_client_test.py --url https://staging-services.paperspace.io/model-serving/de1b8j937fgmkx:predict`
+
+_Note: you may encounter an error about installing Python as a framework related to *matplotlib*. If so, try updating the backend of your matploglib using [the recommendations here](https://stackoverflow.com/questions/21784641/installation-issue-with-matplotlib-python). If that doesn't work, try installing _python3_ with Homebrew and possibly use the [uninstall and reinstall method here](https://github.com/scikit-optimize/scikit-optimize/issues/637).
```

Comment:
Matplotlib is now optional not required. It will simply say that displaying the image wont be possible but otherwise its fully capable/

Reply:
hm, yeah i saw that message but unfortunately even after performing the above commands, i still ran into the issue. i haven't been able to successfully run it on my machine

Reply:
@jaredscheib can you be more specific

Reply:
@jaredscheib i am avaliable today if you could show me that